### ISSUE TITLE
String not localized

### ISSUE THREAD

**Issue by MoltenCoffee** [mikecao/umami#117]
When translating umami to Dutch and testing it, I noticed a string that was not localized. 
When viewing stats of a webpage that has not produced any data yet, and clicking 'more details' on the dashboard, `pages`, `referrers`, `browsers` etc display `No data available`. This string is not included in the language files, and thus cannot be localized.

Edit: the same goes for the country names in the country view, although I suppose this is from an external data source?

**Comment by mikecao** [comment]
You're right, that string is currently in the CSS for empty content. I need to make it a regular string. Good catch.

As for countries, I think it's fine to not localize them since there are so many and would make the localization effort really difficult.

**Comment by mikecao** [comment]
@MoltenCoffee Ok I added the string for `No data available`. If you could update your NL file that would be great. I also noticed `Pages` was translated into `Pagina's`, with the apostrophe. Not sure if a mistake or not.

**Comment by MoltenCoffee** [comment]
I'm not able to send a PR your way right now, but `"Geen gegevens beschikbaar."` would be the translation. If not already added by then, I'll create one tomorrow.

`Pagina's` is the correct spelling! I did notice a typo just now in `metrics.operating-system` though, where I wrote `Besturingssystem` instead of `Besturingssysteem`. Same as above, I could create a PR tomorrow if you haven't already.

Thanks for the quick responses and fixes!
